types/audit.d.ts

```diff
@@ -66,23 +66,25 @@ declare global {
       wastedPercent?: number;
     }
 
+    // TODO: consider making some of the `string | IcuMessage` into just `IcuMessage` to require i18n.
```

Comment:
> is this the main reason we're doing `string | IcuMessage` manually here instead of using the `RawIcu` type?

Yeah, and the ergonomics are also just more straightforward to use, since `RawIcu<>` makes it an indirect step when using in vscode...you can't just see the type on the object, it'll look like `LH.RawIcu<{text: string}>` or `LH.RawIcu<ProductBase>` (and if you click through to `ProductBase` you would only see `string`), so it would only be when accessing the property directly that you would see the type.

(There's [a comment](https://github.com/GoogleChrome/lighthouse/pull/10630/files#diff-ec1b2a8c6b3014e062be782e9bfe51fcR32-R33) on the `RawIcu` type that points to prefer making properties `| LH.IcuMessage` instead of using it for this reason)

I tried making more of these `LH.IcuMessage` only, and I think for every one of them there was some audit that was currently breaking the rules and had a string instead of a localized message, but that was in April, maybe it's better. There's also the question of if these types are just for us, or also for folks writing plugins/custom audits, and they might not want to bother with localization yet, so not allowing `string` would break them.